hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmBucketInfo.java

```diff
@@ -41,6 +42,21 @@ public void protobufConversion() {
         OmBucketInfo.getFromProtobuf(bucket.getProtobuf());
 
     Assert.assertEquals(bucket, afterSerialization);
+  }
+
+  @Test
+  public void testClone() {
+    OmBucketInfo omBucketInfo = OmBucketInfo.newBuilder()
+        .setBucketName("bucket")
+        .setVolumeName("vol1")
+        .setCreationTime(Time.now())
+        .setIsVersionEnabled(false)
+        .setStorageType(StorageType.ARCHIVE)
+        .build();
```

Comment:
Can you add some test with setAcls and check the behavior for reference check TestOmVolumeArgs.

Reply:
Thanks @bharatviswa504 for this comment.
I'm going to update the test and check the behavior .

Reply:
Thanks @bharatviswa504 the advise.
Updated.